serialization/fontDataset.py

```diff
@@ -0,0 +1,89 @@
+import torch
+from torch.utils import data
+import font_pb2
+from os.path import join 
+import numpy as np
+import pandas as pd
+class Dataset(data.Dataset): 
+
+    def __init__(self, proto_dir, glyph_list, dimensions): 
+        '''
+        Parameters
+        ----------
+
+        proto_dir : directory of where all the glyph protos are located
+
+        glyph_list : file path that contains list of glyphs
+        '''
+        self.proto_dir = proto_dir
+        self.glyph_list = pd.read_csv(glyph_list,header=None)
```

Comment:
I don't understand the need for this `glyph_list` file.

Reply:
Because all there are multiple protobufs we need one list for the dataloader to query from. 
I got it from an example online

Reply:
But if we separate each glyph into its own (sharded) proto, there won't be a need for this csv, right?